main/FatesInventoryInitMod.F90

```diff
@@ -938,13 +915,12 @@ subroutine set_inventory_edcohort_type1(csite,bc_in,css_file_unit,npatches, &
       integer,  parameter :: recruitstatus = 0
 
      
-      read(css_file_unit,fmt=*,iostat=ios) c_time, p_name, c_name, c_dbh, c_height, &
-            c_pft, c_nplant, c_bdead, c_balive, c_avgRG
+      read(css_file_unit,fmt=*,iostat=ios) c_time, p_name, c_dbh, &
+            c_height, c_pft, c_nplant
```

Comment:
I guess c_name is not really critical, but it may be a good idea to keep it as a way of indexing the cohorts to the original trees in an inventory (e.g., the tree tag number).

Reply:
My preference is to remove columns not used. If inventory data changes then people can remake the FATES files which wouldn't require indexing by tree tag. But I don't feel too strongly about this. I could put it back if people would miss it.